tasks/oah-install.yml

```diff
@@ -0,0 +1,35 @@
+---
+
+- name: Install python-docker requirements
+  become: yes
+  ansible.builtin.apt:
+    name:
+      - python3-docker
+    state: latest
+
+- name: Check if user exist in docker group
+  ansible.builtin.shell: |
+    grep -q /etc/group -e "docker" 
+  register: result
+
+- name: Adding user to docker group
+  become: yes
+  shell: |
+    gpasswd -a $USER docker
+    newgrp docker
+  when: result.rc != 0
+
+- name: Make sure docker is started
+  service:
+    name: docker
+    state: started
+
+- name: Pulling fossology image
```

Comment:
Getting an error while pulling fossology image
- Installed latest version of ansible > 2.7.1(module required_version)
- Docker service is running
- Installed community.docker collection

![image](https://user-images.githubusercontent.com/36243506/190916729-66ef3c9d-1cf6-414f-8d29-1bb3490fd7c2.png)

Reply:
Code : 
![image](https://user-images.githubusercontent.com/36243506/190916786-f36a80c1-6972-43c7-a4f4-f3db34f2225e.png)

Reply:
Code : 
![image](https://user-images.githubusercontent.com/36243506/190916819-ef2e6a78-8be2-4b26-a0dc-625b3395d228.png)

Reply:
@Pranshu021 Can you please try installing the latest version for ansible using this [link](https://www.cyberciti.biz/faq/how-to-install-and-configure-latest-version-of-ansible-on-ubuntu-linux/).
![image](https://user-images.githubusercontent.com/58501088/190920670-2a5e6c83-63f0-405d-b16e-de40e38eca77.png)

Reply:
community.docker.docker_image module is running successfuly now after installing latest ansible-core version, but this collection does not come pre-installed with ansible-core - https://docs.ansible.com/ansible/latest/collections/community/docker/docker_image_module.html#parameter-state

Maybe (if required) put a task above this to run collection install command - `ansible-galaxy collection install community.docker`

Otherwise, playbook install.yml running successfuly !